cypress/e2e/Taxdata/APIinfo.cy.js

```diff
@@ -0,0 +1,30 @@
+describe('Taxdata APIs info page ', () => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
why do we need this?

Reply:
There were some cases where the uncaught exception was throwing from cypress end due to which other test cases were also failing. so that we are using this.